libcxx/include/algorithm

```diff
@@ -422,11 +422,12 @@ namespace ranges {
   template<random_access_iterator I, sentinel_for<I> S, class Comp = ranges::less,
           class Proj = identity>
     requires sortable<I, Comp, Proj>
-    I ranges::stable_sort(I first, S last, Comp comp = {}, Proj proj = {});                 // since C++20
+    constexpr I                                                                             // constexpr in C++26
```

Comment:
For consistency, the same for the other overload in this header.
```suggestion
    constexpr I                                                                             // constexpr since C++26
```

Reply:
Hmm, existing comments in `<algorithm>` say "constexpr in C++14/20/26". Should we change all of them to use "since"?

Reply:
Yes please, we use since in other places too.

Reply:
Changed.